DependencyInjection/EkinoNewRelicExtension.php

```diff
@@ -39,22 +39,23 @@ public function load(array $configs, ContainerBuilder $container)
             $loader->load('block.xml');
         }
 
-        $container->setParameter('ekino.new_relic.request_listener.ignored_routes', $config['ignored_routes']);
-        $container->setParameter('ekino.new_relic.request_listener.ignored_paths', $config['ignored_paths']);
-        $container->setParameter('ekino.new_relic.request_listener.ignored_commands', $config['ignored_commands']);
+        $container->getDefinition('ekino.new_relic.request_listener')
+            ->replaceArgument(2, $config['ignored_routes'])
+            ->replaceArgument(3, $config['ignored_paths']);
+
+        $container->getDefinition('ekino.new_relic.command_listener')
+            ->replaceArgument(2, $config['ignored_commands']);
 
         $interactor = $config['enabled'] && extension_loaded('newrelic')
             ? 'ekino.new_relic.interactor.real'
             : 'ekino.new_relic.interactor.blackhole';
 
-        if ($config['logging'])
-        {
+        if ($config['logging']) {
             $container->setAlias('ekino.new_relic.interactor', 'ekino.new_relic.interactor.logger');
             $container->getDefinition('ekino.new_relic.interactor.logger')
                 ->replaceArgument(0, new Reference($interactor));
         }
-        else
-        {
+        else {
```

Comment:
shouldn't it be `} else {`?

Reply:
Yeah, True. I was not supposed to do CS fixes but that one really hurt my eyes. =)

Thanks.

Reply:
```
}
else {
```

hurts much more my eyes ;-)

What's about adding a php-cs-fixer with basic rules?

Reply:
=)

Excellent idea, I've created a PR: https://github.com/ekino/EkinoNewRelicBundle/issues/142